c/json.c

```diff
@@ -362,8 +385,8 @@ void jsonConvertAndWriteBuffer(jsonPrinter *p, char *text, size_t len,
       DUMPBUF(text, len);
       newLen = convertToUtf8(p, len, text, inputCCSID);
       if (newLen < 0) {
-        ERROR("jsonConvertAndWriteBuffer() error: newLen = %d\n",
-                newLen);
+        JSONERROR("jsonConvertAndWriteBuffer() error: newLen = %d\n",
```

Comment:
I think `%zu` is used with `size_t`.

Reply:
Does zu actually work on xlc 2.4 (not xlclang or clang)??

Reply:
Yes, I've used it in XLC.